src/clj/game/cards-ice.clj

```diff
@@ -336,9 +337,10 @@
                                       (->> (vec (.split (:subtype card) " - "))
                                            (concat [target])
                                            (join " - "))))
+                      (update! (assoc card :subtype-target target))
```

Comment:
Aha! This makes sense. Is this why I see `(get-card state card)` all over the place?